Makefile

```diff
@@ -27,14 +27,15 @@ help: Makefile
 testDataDir := test/data/
 tempDir := ${testDataDir}temp/
 gitDataDir := ${tempDir}sdk-test-data/
-branchName := main
+branchName := sameeran/ff-3687-obfuscated-precomputed-json
```

Comment:
Temporarily using this branch for testing until https://github.com/Eppo-exp/sdk-test-data/pull/89 gets an approval 🙏